docs/src/common.md

```diff
@@ -218,57 +224,62 @@ nothing # hide
 
 We plot first the value of the Jacobi constant as function of time.
 ```@example common
-plot(solT.t, H.(solT.u), label="TaylorIntegration.jl")
+plot(solT.t, H.(solT.u), label="TaylorMethod(25)", fmt = :png, yformatter = :plain)
 plot!(solV.t, H.(solV.u), label="Vern9()")
 xlabel!("t")
 ylabel!("H")
 ```
-Clearly, the integration with `Vern9()` does not conserve the Jacobi constant;
-actually, the fact that its value is strongly reduced leads to the artificial trapping
-displayed above around ``m_1``. We notice that the loss of conservation of the
-Jacobi constant is actually not related to a close approach with ``m_1``.
+In the scale shown we observe that, while both solutions
+display a preservation of the Jacobi constant to a certain degree, the `Vern9()`
+solution suffers sudden jumps during the integration.
 
 We now plot, in log scale, the `abs` of the absolute error in the Jacobi
 constant as a function of time, for both solutions:
 ```@example common
-plot(solT.t, abs.(δET), yscale=:log10, label="TaylorIntegration.jl")
+plot(solT.t, abs.(δET), yscale=:log10, label="TaylorMethod(25)", legend=:topleft, fmt = :png, yformatter = :plain)
 plot!(solV.t, abs.(δEV), label="Vern9()")
-ylims!(10^-18, 10^4)
+ylims!(10^-16, 10^-10)
 xlabel!("t")
 ylabel!("dE")
 ```
-We notice that the Jacobi constant absolute error for the `TaylorIntegration.jl`
-solution remains bounded below ``5\times 10^{-14}``, despite of the fact that
-the solution displays many close approaches with ``m_2``.
+We notice that the Jacobi constant absolute error for the `TaylorMethod(25)`
+solution remains bounded below ``2\times 10^{-14}``, while the `Vern9()` solution
+displays sudden jumps earlier in the integration and has variations larger than
+``10^{-13}``.
```

Comment:
Our solution display changes in the Jacobi constant below 10^(-13); the Vern9 solution is, at the end of the integration time, two order of magnitude above our solution for the same quantity. This  is also worth mentioning.

Reply:
Sure, updated!